### PR TITLE
【Hackathon 6th Fundable Projects 3 No.76】fluid operator crf_decoding

### PR DESCRIPTION
**Pull request by co63oc**
### PR Category

Others 

### PR Types

Others 

### Description

迁移  crf_decoding

### CODE REVIEW

**Comment by co63oc** [comment]
@YuanRisheng CI已完成需要review下

**Review by YuanRisheng — DISMISSED** [review]

**Review by wanghuancoder — DISMISSED** [review]
LGTM

File: paddle/phi/infermeta/multiary.cc [hunk 1]
```diff
@@ -1101,6 +1101,116 @@ void ConcatInferMeta(const std::vector<const MetaTensor*>& x,
...
+              label_dims.size(),
+              label_dims));
+    }
+  }
+
+  viterbi_path->share_lod(emission);
+  if (has_length) {
+    viterbi_path->set_dims({emission_dims[0], emission_dims[1]});
+  } else {
+    viterbi_path->set_dims({emission_dims[0], 1});
```
Comment (zyfncg) [inline paddle/phi/infermeta/multiary.cc:1210]: data_type也设置下

Comment (co63oc) [reply paddle/phi/infermeta/multiary.cc:1210]: 已修改

**Review by zyfncg — APPROVED** [review]

**Pull request merged**